dss-document/src/main/java/eu/europa/esig/dss/validation/DiagnosticDataBuilder.java

```diff
@@ -806,7 +806,7 @@ private XmlCertificate getXmlCertificate(Set<DigestAlgorithm> usedDigestAlgorith
 		final XmlCertificate xmlCert = new XmlCertificate();
 
 		xmlCert.setId(certToken.getDSSIdAsString());
-                xmlCert.setBase64Encoded(Utils.toBase64(certToken.getEncoded()).getBytes());
```

Comment:
I removed this, because it seems to encode twice to base64 (once here + once under exporting the report itself).